### PR TITLE
Add Army Field Maintenance's recipes to Under the Hood

### PR DESCRIPTION
**Pull request by ISuckM8**
#### Summary

Balance "Add Army Field Maintenance's recipes to Under the Hood"

#### Purpose of change

The recipes the manual provides are extremely simple and realistically don't need a custom manual.

#### Describe the solution

Simply make the recipes available when you have 2 in mechanics through Under the Hood in addition to the army field maintenance manual.

#### Describe alternatives you've considered

Making the recipes require other manuals too.

#### Testing

It's working just fine.

#### Additional context

Nothing here.

### CODE REVIEW

File: gfx/UltimateCataclysm/tile_config.json [hunk 1]
```diff
@@ -258,7 +258,6 @@
             "mag_smg",
             "mag_archery",
             "mag_cooking",
-            "mag_fieldrepair",
```
Comment (HadeanLake) [inline gfx/UltimateCataclysm/tile_config.json:261]: tile.config.json files are generated from files in tilesets repository, you want to remove it there instead:
https://github.com/I-am-Erk/CDDA-Tilesets/blob/6c67be12839e364498099fae945e661323b7f69e/gfx/UltimateCataclysm/pngs_small_20x20/items/books/texts.json#L106

**Comment by Maleclypse** [comment]
Make them recipe level one in the army field repair guide. Leave them level 2 in Under the Hood.  So if you get lucky you can learn them early with the book designed for teaching people with no mechanical ability.

**Comment by ISuckM8** [comment]
> Make them recipe level one in the army field repair guide. Leave them level 2 in Under the Hood. So if you get lucky you can learn them early with the book designed for teaching people with no mechanical ability.

Done.

**Comment by RenechCDDA** [comment]
Is there any particular reason to delete the army manual? AFAIK it's a real manual, FM 20-22 "VEHICLE RECOVERY OPERATIONS".
image

**Comment by ISuckM8** [comment]
> Is there any particular reason to delete the army manual? AFAIK it's a real manual, FM 20-22 "VEHICLE RECOVERY OPERATIONS". image

I didn't want to have a fairly common manual that has every recipe an advanced one does, but since Maleclypse wants it I'll just keep it.

**Comment by kevingranade** [comment]
I am 100% certain that normal mechanics references do not have things like field expedient replacements for belts and filters.

I also don't agree with your assertion that these recipes are trivial for anyone that isn't very experienced with working on cars.  I can certainly see someone thinking they are and trying *something*, then Finding Out as the phrase goes, but that is not a thing we support in the game.

**Pull request closed without merging**